.env

```diff
@@ -0,0 +1,2 @@
+VITE_BASE_URL = https://target-api-induction-v2.herokuapp.com/api/v1/users
+VITE_SIGN_IN =/sign_in
```

Comment:
I don't think you should include the `/users` in the api url (not all routes have that path) and add an environment variable for the `/sign in` route. Also, remember to add this file to gitignore if you then add some key

Reply:
Perfect, I've updated them accordingly. And also added the .env to git ignore, only left an .env.example